static/panes/tree.ts

```diff
@@ -445,6 +452,13 @@ export class Tree {
 
         dragSource.on('click', () => {
             this.hub.addInEditorStackIfPossible(dragConfig.bind(this));
+            // at this point the editor is initialized with default contents
```

Comment:
This added code doesn't make much sense to me. This is not the only place where you can add editors and at this point there is no actual physical editor yet which you can query. What does this solve what the onEditorOpen cannot catch?

Reply:
`onEditorOpen` was indeed the first choice for this code, but during its execution the `file` retrieved by `multifileService.getFileByEditorId` has no `filename` and no `content`. These are set by the `Editor` ctor, a bit later.

Reply:
@partouf  LGTU?

Reply:
I will go ahead and merge, can fix later whatever comes up.